easybuild/easyconfigs/i/icc/icc-2019.4.227-GCC-8.3.0.eb

```diff
@@ -0,0 +1,32 @@
+# This is an easyconfig file for EasyBuild, see http://easybuilders.github.io/easybuild
+
+name = 'icc'
+version = '2019.4.227'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "Intel C and C++ compilers"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15538/']
+sources = ['parallel_studio_xe_%(version_major)s_update%(version_minor)s_composer_edition_for_cpp.tgz']
+checksums = ['09414c51def0b0ad0611d5560bfdce51bd70a53bc6934b5709d81ec0d055aa34']
+
+local_gccver = '8.3.0'
+versionsuffix = '-GCC-%s' % local_gccver
```

Comment:
How about getting rid of the `GCC` vversion suffix? It is quite redundant since there is only one `GCC` in the common toolchains

Reply:
Because that icc version (2019.4.227) could be built against multiple GCC versions locally.

Reply:
@akesandgren Couldn't we say that about basically all modules and almost all packages? That someone *could* make a custom easyconfig locally that requires a suffix.

Reply:
I think it makes sense to drop the versionsuffix, since we stick to a single GCC version anyway.

If a need arises to install the Intel compilers on top of a different GCC version, then that can be done by re-introducing the `versionsuffix`?

Reply:
+1
We haven't had a versionsuffix locally since the beginning of the CC stack and it hasn't caused any issue (we haven't had any reason so far to pair a single iccifort with multiple GCC's so far).